src/components/subject-icons/pages/subjects-interactive.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import SubjectIcon, {TYPE, SIZE} from '../SubjectIcon';
+import SubjectIconBox from '../SubjectIconBox';
+import DocsActiveBlock from 'components/DocsActiveBlock';
+
+const Subjects = () => {
+  const settingsIcons = [
+    {
+      name: 'type',
+      values: TYPE
```

Comment:
add `required: true` otherwise there will be an (invalid) option "(optional)" in the dropdown